service/src/main/java/com/fecklessweasel/service/objectmodel/Department.java

```diff
@@ -68,6 +72,36 @@ public static Department create(Connection conn, University university, String d
     }
 
     /**
+     *@Author Hayden Schmackpfeffer
+     *@param connection the SQL Connection
+     *@param univ University object model
+     *@return all departments belonging to passed in university
+     */
+    public static ArrayList<Department> getAllInUniversity(Connection conn, University univ)
```

Comment:
Should we have Department.getAllInUniversity() or would it make more sense to have (alternatively or in addition) a University.getAllDepartments() method.